src/lib/components/AppBar.svelte

```diff
@@ -20,7 +20,7 @@
       on:keypress={toggleMenu}
     >
       <li>
-        <a id="app" href="https://app.blendreading.com" target="_blank" rel="noopener noreferrer">app</a>
+        <a id="app" href="https://app.blendreading.com" target="_blank" rel="noopener noreferrer" on:click={ () => gtag('event', 'open_app') }>app</a>
```

Comment:
This may not reliably fire the event before navigating. When you have an `onclick` attribute on an `a` tag, the protocol is:
```
1. Run the `onclick` function
2. If the function returns `false`, cancel the navigation
3. Else, navigate to `href`
```
Here, your function will run, call the `gtag` function which will trigger a fetch request to Google (not immediately, it just [adds an element](https://github.com/CSMA-Technology/blend-product-site/blob/main/src/routes/%2Blayout.svelte#L8) to an array and I think it overrides the `push` method so that it will trigger a network request, probably after a debounce). But nothing will wait for that fetch request, so it is just a race between that and the browser navigating to the next page. 

If we want to fire the event reliably, we'd have to use the [event_callback](https://developers.google.com/tag-platform/gtagjs/reference/parameters) parameter to the `gtag` function. i.e.:
```suggestion
        <a id="app" href="https://app.blendreading.com" target="_blank" rel="noopener noreferrer" on:click={ () => {gtag('event', 'open_app', { event_callback: () => window.location.href="app.blendreading.com"}; return false;)} }>app</a>
```

The tradeoff here is obviously on a slow connection or some problem with `gtag`, navigation will be delayed. But in those scenarios the event would not have been fired probably, so we just have to decide if that is okay or not.